lib/sheet.js

```diff
@@ -0,0 +1,35 @@
+import Spreadsheet from 'edit-google-spreadsheet';
+import googleAuth from './google-auth';
+import util from './util';
+
+class Sheet {
+
+    requestSpreadsheet(worksheetId, spreadsheetId, callback){
+        googleAuth.authenticate(tokens => {
+            Spreadsheet.load({
+                debug: (process.env.NODE_ENV === 'TEST' ? true : false), 
```

Comment:
`booleanCondition ? true : false` is superfluous. only `booleanCondition` will do (and save you an extra comparison).